code/modules/industrial_lift/elevator/elevator_panel.dm

```diff
@@ -64,6 +69,9 @@
 	// this is just here for redundancy's sake.
 	. = INITIALIZE_HINT_LATELOAD
 
+	if(elevator_music_toggle)
+		elevator_music = new(src, start_immediately = TRUE)
```

Comment:
I'm surprised this works without that comment trick, usually this completely borks with `new`.

Reply:
It just does 🤷